concurrency-limits-grpc/src/test/java/com/netflix/concurrency/limits/grpc/util/StringMarshaller.java

```diff
@@ -1,4 +1,4 @@
-package com.netflix.concurrency.limits.grpc;
+package com.netflix.concurrency.limits.grpc.util;
```

Comment:
This class is public, so changing the package is a breaking change. If this is just code cleanup, let's move it back and avoid the break.

Reply:
This is a test jar, shouldn't be an issue.